python/apps/redis-to-postgres/tests/test_config.py

```
@@ -0,0 +1,90 @@
+[worker.worker-1.observability]
+prometheus_port = 8012
+service_name    = "redis-to-postgres-worker-1"
+trace_endpoint  = "http://localhost:4318/v1/traces"
+
+[worker.worker-2.observability]
+prometheus_port = 8013
+service_name    = "redis-to-postgres-worker-2"
+trace_endpoint  = "http://localhost:4318/v1/traces"
+'''
+
+def parse_config_from_text(config_text: bytes) -> AppConfig:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)